GTG/core/datastore.py

```diff
@@ -281,7 +281,7 @@ def do_first_run_versioning(self, filepath: str) -> None:
             self.first_run(self.data_path)
 
 
-    def find_old_path(self, datadir: str) -> None | str:
+    def find_old_path(self, datadir: str) -> Union[None, str]:
```

Comment:
`Optional[str]` would work here as well - no need to import Union, actually.

Longer-term, I'd love to avoid `Optional[str]` and just use an empty string as the default value.

Reply:
Uh, nice! Good catch!